cmake/linux/package_linux.sh.in

```diff
@@ -80,15 +80,19 @@ success "${APPDIR}"
 
 # Create a wrapper script which calls the lmms executable
 mv "$APPDIR/usr/bin/lmms" "$APPDIR/usr/bin/lmms.real"
-echo "#!/usr/bin/env bash
-DIR=\"\$( cd \"\$( dirname \"\${BASH_SOURCE[0]}\" )\" && pwd )\"
+# shellcheck disable=SC1083
```

Comment:
@zapashcanon I couldn't find a way around this.  `shellcheck` can't see the quotes when it's a `cat> file ... <<EOL` syntax, but what I'm reading, this approach is more common versus the `echo ... > file` syntax.  It also requires less escaping, so it's slightly easier to read.  Thoughts welcome.